app/Console/Command/FindMatchingTrailersOnYouTube.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Json;
+use CultuurNet\UDB3\Kinepolis\Trailer\TrailerRepository;
+use CultuurNet\UDB3\Model\ValueObject\Identity\Uuid;
+use CultuurNet\UDB3\Offer\Commands\Video\AddVideo;
+use CultuurNet\UDB3\ReadModel\DocumentDoesNotExist;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\ReadModel\JsonDocument;
+use CultuurNet\UDB3\Search\ResultsGenerator;
+use CultuurNet\UDB3\Search\ResultsGeneratorInterface;
+use CultuurNet\UDB3\Search\SearchServiceInterface;
+use CultuurNet\UDB3\Search\Sorting;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class FindMatchingTrailersOnYouTube extends Command
```

Comment:
It does more than just searching a movie on YouTube, it also adds a matching trailer. Maybe rename the class to something like `AddMatchingTrailer`?

Because an interface `TrailerRepository` is used it doesn't need to be bound to YouTube?